comm/src/main/scala/coop/rchain/comm/UPnP.scala

```diff
@@ -133,7 +133,7 @@ object UPnP {
     for {
       _       <- Log[F].info("trying to open ports using UPnP....")
       devices <- Sync[F].delay(discover)
-      res <- if (devices.gateways.isEmpty) logGatewayEmpty(devices) *> None.pure[F]
+      res <- if (devices.gateways.isEmpty) logGatewayEmpty(devices) >> None.pure[F]
```

Comment:
What's the reason for the change to `>>`?

Reply:
@tymm Because Applicative `productR` a.k.a `*>` is considered dangerous. It can lead to memory leaks and/or race conditions. However, this change is not related to the PR ticket. I used the opportunity to clean up and replace `productR` with `flatMap` in the `comm` layer.

(see: https://twitter.com/impurepics/status/1094339310571175937)